src/R/Support/Impl/Help/Packages/PackageIndex.cs

```diff
@@ -201,38 +200,55 @@ public sealed class PackageIndex : IPackageIndex {
             }
         }
 
+        private async Task UpdateInstalledPackagesAsync() {
+            var token = await _buildIndexLock.WaitAsync();
+            try {
+                var installed = await GetInstalledPackagesAsync();
+                var installedNames = installed.Select(p => p.Package).Append("rtvs");
+
+                var currentNames = _packages.Keys.ToArray();
```

Comment:
Since it is concurrent bag, I wanted to capture its current, rather than changing, state.